test/gateway/ost_prime/increase_supply.js

```diff
@@ -64,23 +64,6 @@ contract('OSTPrime.increaseSupply()', function (accounts) {
 
   });
 
-  it('should fail when account address is zero', async function () {
-
-    await initialize();
-
-    beneficiary = NullAddress;
-
-    await Utils.expectRevert(
-      ostPrime.increaseSupply(
-        beneficiary,
-        amount,
-        { from: coGatewayAddress },
-      ),
-      'Account address should not be zero.',
-    );
-
-  });
-
```

Comment:
should it pass on zero address (burning)? does that "increase supply"?

Reply:
No, for burning it should transfer token to the burner address and which should redeem and unstake at zero address on origin chain.

Reply:
Then why did you remove the test?